README.md

```diff
@@ -46,6 +46,11 @@ or if you prefer you can run this in the console:
 $ curl -sL https://civo.com/get | sh
 ```
 
+### Installing on windows
+First you need [scoop](https://scoop.sh/) installed then:
```

Comment:
I think that it might be a good idea to note that unzipping the windows-amd64 file in the releases here in the repository is a valid way too (although it have to be added to PATH in one or another way, hehe).